### PR TITLE
fix: ratecounterserver exits

### PR DESCRIPTION
**Pull request by chasers**
RateCounterServer is failing to start with an already_started error which is putting the RecentLogServer in a crash loop.

### CODE REVIEW

File: lib/logflare/source/recent_logs_server.ex [hunk 1]
```diff
@@ -240,11 +241,18 @@ defmodule Logflare.Source.RecentLogsServer do
     end
   end
 
+  def handle_info({:EXIT, _from, _reason}, state) do
```
Comment (Ziinc) [inline lib/logflare/source/recent_logs_server.ex:244]:
```suggestion
  def handle_info({:EXIT, _from, _reason}, _state) do
```

**Review by Ziinc — APPROVED** [review]

**Comment by Ziinc** [comment]
would need to fix checks as well

File: lib/logflare/source/recent_logs_server.ex (hunk 1 above)
Comment (chasers) [reply lib/logflare/source/recent_logs_server.ex:244]: Huh? Don't I need state in my :noreply tuple?

Comment (chasers) [reply lib/logflare/source/recent_logs_server.ex:244]: Maybe you were meaning the terminate...

**Review by hauleth — APPROVED** [review]

**Pull request merged**